### DISCUSSION TITLE
Ability to add folder in sidebar without making a page

### DISCUSSION THREAD

**Discussion by joezappie** [outline/outline#3591] [Ideas]
Currently, if you want to organize the sidebar into sub folders, you have to create a page. In my setup, this leads to dozens of blank pages that only have a title as I have lots of nested data for different projects and departments.

I would be nice to be able to make a "folder" without having to have a page attached to it. When clicking on it, its only behavior would be to expand or collapse the group instead of change the page.

**Comment by almereyda** [comment]
Eventually we'd want to call such kind of folder a "sub collection", as in: we get a nice overview list of documents, and can then still choose to voluntarily provide a small description, when needed?

**Reply by joezappie** [reply]
Ahh yes, that makes more sense than what I proposed. Act the same as a top level collection, and even being able to specify icons for it would be handy to differentiate it from a document.

**Reply by almereyda** [reply]
Yes, icons work very well when using Unicode Emojis in document titles, too.

The available set is a different one.

![Bildschirmfoto von 2022-05-30 01-59-35](https://user-images.githubusercontent.com/1645308/170896721-342492f1-2da8-4b63-a0bd-42cb8fdf5b3b.png)

**Comment by n8iac** [comment]
This is the only feature missing for me from Outline that would really allow for more granular organization of content. Would love to see this implemented.
